src/analog/commands/DataSegmentBase.ts

```diff
@@ -50,4 +53,13 @@ export default class DataSegmentBase extends Command {
 
         return new this(buffer.getDataSegment());
     }
+
+    toJson ( encodeFormat: TBytesEncodeFormat = HEX, hexFormat: IHexFormatOptions = {} ) {
```

Comment:
interface is ambiguous

it allows to choose `base64` but still configure hex options

Reply:
fixed